src/behavior.js

```diff
@@ -35,5 +35,17 @@ Marionette.Behavior = Marionette.Object.extend({
   proxyViewProperties: function (view) {
     this.$el = view.$el;
     this.el = view.el;
+  },
+
+  // First emit on the behavior, then emit on the view context
+  triggerMethod: function() {
```

Comment:
I have definitely wanted this before. In my case, I had a special type of view (a grid), that has a special action (show/hide columns in the grid). Using grid-specific behaviors, I would like those behaviors to act on the grid. Currently, I have to do 
```js
// in the behavior
this.triggerMethod('columns:show')
this.view.triggerMethod('columns:show')
```